src/Native/Utils.js

```diff
@@ -357,6 +357,10 @@ Elm.Native.Utils.make = function(localRuntime) {
 
 	var toString = function(v)
 	{
+		if (v === null)
+			{
+				return '<internal structure>';
+			}
```

Comment:
As far as I can see the source and my fix are both tab indented.

`undefined` is handled well with the existing code, so it seems sufficient just to test for `null`.